ipykernel/zmqshell.py

```diff
@@ -378,7 +378,7 @@ def connect_info(self, arg_s):
         print ("Paste the above JSON into a file, and connect with:\n"
             "    $> jupyter <app> --existing <file>\n"
             "or, if you are local, you can connect with just:\n"
-            "    $> jupyter <app> --existing {0}\n"
+            f"    $> jupyter <app> --existing {connection_file}\n"
             "or even just:\n"
             "    $> jupyter <app> --existing\n"
             "if this is the most recent Jupyter kernel you have started.".format(
```

Comment:
You also need to remove `.format(connection_file)`.

Reply:
it was removed, like almost all other format calls in next commit

Reply:
Thanks!